src/components/Calculator.js

```diff
@@ -1,27 +1,16 @@
 import React, { useState } from 'react';
+import calculate from '../logic/calculate';
 
-function App() {
-  const [input, setInput] = useState('');
+function Calculator() {
+  const [calculatorData, setCalculatorData] = useState({
+    total: null,
+    next: null,
+    operation: null,
+  });
 
-  const handleClick = (value) => {
-    if (value === '=') {
-      try {
-        // eslint-disable-next-line
-                setInput(eval(input).toString());
-      } catch (error) {
-        setInput('Error');
-      }
-    } else if (value === 'C') {
-      setInput('');
-    } else if (value === 'AC') {
-      setInput('');
-    } else if (value === '+/-') {
-      setInput((prevInput) => (prevInput.charAt(0) === '-' ? prevInput.slice(1) : `-${prevInput}`));
-    } else if (value === 'Del') {
-      setInput((prevInput) => prevInput.slice(0, -1));
-    } else {
-      setInput(input + value);
-    }
+  const handleClick = (buttonName) => {
```

Comment:
Good job working on the project requirements. I noticed you disabled eslint in this file line number 58 it is against microverse policy to disable linters because linters help us write clean code. It would be nice if you enabled eslint. 

This is your code please take a look at this screen shot:


![Screenshot 2023-09-13 at 6 46 42 PM](https://github.com/Hsuwaddyhtat22/react-magician/assets/79506105/1e95d729-5f4e-4222-b74b-1385e97d6dcb)